examples/cookbook/networking/fetch_data/lib/main.dart

```diff
@@ -36,11 +36,19 @@ class Album {
   });
 
   factory Album.fromJson(Map<String, dynamic> json) {
-    return Album(
-      userId: json['userId'] as int,
-      id: json['id'] as int,
-      title: json['title'] as String,
-    );
+    return switch (json) {
+      {
+        'userId': int userId,
+        'id': int id,
+        'title': String title,
+      } =>
+        Album(
+          userId: userId,
+          id: id,
+          title: title,
+        ),
+      _ => throw 'Failed to load album',
```

Comment:
[`FormatException`](https://api.dart.dev/stable/dart-core/FormatException-class.html) is made for this situation.

```suggestion
      _ => throw FormatException('Failed to load album.'),
```

However I expected this to trigger a lint. What do you think about enabling [`only_throw_errors`](https://dart.dev/lints/only_throw_errors)?